### DISCUSSION TITLE
Analyse local directory

### DISCUSSION THREAD

**Discussion by gitalm** [mcguirepr89/BirdNET-Pi#850] [Ideas]
Dear @mcguirepr89 ,

thanks for the wonderful BirdNetPi. I have also sound files from Mobiles and other Audio Recorder. Would it be possible to just point the BirdNetPi to a directory instead of a Microphone?
The analysis is tremendous and so all the work could be used for other wav-files too.

Best wishes Johannes

**Comment by ampsu** [comment]
I believe the easiest way is to just use BirdNET-Analyzer on a desktop. It can analyze the whole local folder and output results to txt files.
I think that BirdPi cant really provide you any useful insights since it requires correctly named files?

**Reply by ehpersonal38** [reply]
This is a good idea, BirdNET-Analyzer would provide the basic functionality that BirdNET-Pi does, it just won't analyze your files on a 24/7 basis unless you write some sort of cronjob to constantly call the analyze.py script. It also won't write detections to a single database, but rather to multiple TXT or CSV files.

If you wanted to use BirdNET-Pi, it will actually automatically pick up and start analyzing any .wav file that is placed in `~/BirdSongs`. You'll just have to make sure the .wav file follows a particular naming format otherwise the server will reject it. You can find that format in birdnet_recording.sh: https://github.com/mcguirepr89/BirdNET-Pi/blob/main/scripts/birdnet_recording.sh#L53

**Reply by Svardsten53** [reply]
Another solution is to have some kind of player and connect its speaker output to the sound card of the BirdNET-Pi computer instead of a microphone. If you then do "Clear All data" and set the date and time manually in settings (untick the NTP update), you will get a completely normal report with database, audio files, statistics and more for the audio files you play. I myself have tested this several times.

**Reply by ampsu** [reply]
I guess it is also possible to use stereo mix as a source? So you can just create a playlist and play those files. Actually I am note sure if Pi supports stereo mix like as Windows does but maybe? :D
